### DISCUSSION TITLE
Support long datatype in writing hdf5 file

### DISCUSSION THREAD

**Discussion by lizhang386b** [jamesmudd/jhdf#587] [General]
First of all thank you @jamesmudd for creating this project. It makes java programming a lot easier to create/read hdf5 files.
It seems long datatype is not supported yet in writing. Any plan on when this feature will be added and released?

**Comment by jamesmudd** [comment]
Thanks for your interest in jHDF. Your right writing `long` dataset is not yet supported. This will be added in the next release hopefully not too long away, just want to clean up the code for writing attributes.

**Comment by jamesmudd** [comment]
Should be working now in [v0.8.0](https://github.com/jamesmudd/jhdf/releases/tag/v0.8.0) let me know. Thanks

**Comment by lizhang386b** [comment]
Confirmed 0.8.0 works with long type in writing HDF5.
Thank you very much for the fast release. Appreciate it.
